instructor_admin/ui/Text/Translations/TextWord.elm

```diff
@@ -141,86 +152,83 @@ new :
     -> WordKind
     -> Endpoints
     -> TextWord
-new id section instance phrase grammemes translations word endpoints =
-    TextWord id section instance phrase grammemes translations word endpoints
+new wordId section inst _ maybeGrammemes maybeTranslations word endpnts =
+    TextWord wordId section inst phrase maybeGrammemes maybeTranslations word endpnts
 
```

Comment:
Looks like we are throwing away the phrase param here with `_`. `Text.Translations.TextWord.new`  will still need to use a `phrase` param in order to construct a `TextWord`.  Maybe rename `_` here to `phr` so it doesn't conflict with the `phrase` function (not great but works).

Reply:
Will do.